src/client/debugger/jupyter/debuggingManager.ts

```diff
@@ -234,22 +244,40 @@ export class DebuggingManager implements IExtensionSingleActivationService, IDeb
         this.runByLineInProgress.set(runningByLine).ignoreErrors();
     }
 
-    private async startDebugging(doc: NotebookDocument, cell?: NotebookCell, options?: DebugSessionOptions) {
+    private async startDebuggingCell(doc: NotebookDocument, mode: KernelDebugMode.Cell | KernelDebugMode.RunByLine, cell: NotebookCell) {
+        const config: IKernelDebugAdapterConfig = {
+            type: pythonKernelDebugAdapter,
+            name: path.basename(doc.uri.toString()),
+            request: 'attach',
+            internalConsoleOptions: 'neverOpen',
+            justMyCode: true,
+            // add the doc uri to the config
+            __document: doc.uri.toString(),
```

Comment:
@DavidKutu is this used anywhere? I couldn't actually find a usage

Reply:
Yeah, I guess I removed it. We send the NotebookDocument as a parameter anyway. I'll remove it on my next PR.